explore.css

```diff
@@ -0,0 +1,828 @@
+    border: 0;
+    border-radius: 15px;
+    display: grid;
+    grid-template-rows: 30px 70px 70px 70px 80px;
+
+
+}
+.follow-suggestion > div:hover{
+    background-color:#D3D3D3;
+    cursor: pointer;
+}
+.follow-suggestion > p {
+    font-size: 15px;
+    align-self: center;
+    color: rgb(57, 156, 192);
+    
+}
+.follow-suggestion > p:hover{
+    cursor: pointer;
+}
+.follow-suggestion > div{
+    display: grid;
+    grid-template-columns: 50px 1fr 80px;
+    padding: 10px;
+}
+.follow{
+    display: grid;
+    grid-template-rows: 20px 20px;
+}
+.follow > p{
+    font-size: 15px;
+}
+.follow >p:nth-of-type(2){
+    font-size: 15px;
+    color:rgb(152, 141, 141)
+}
+.follow > p > img{
+    height: 15px;
+    width: 15px;
+    color: #2873ec;
+    filter: invert(37%) sepia(91%) saturate(2685%) hue-rotate(206deg) brightness(94%) contrast(97%);
+}
+.follow-suggestion > h2{
+    font-size: 20px;
+}
+.follow-suggestion > div > button{
+    font-size: 15;
+    width: 78px;
+    height: 32px;
+    background-color: black;
+    color: white;
+    border: 0;
+    border-radius: 50px;
+}
+.follow-suggestion > div > img{
+    width: 48px;
+    height: 48px;
+    border-radius: 50px;
+    object-fit: cover;
+}
+/* MEDIA */
+
+@media screen and (max-width: 1265px) {
+    .topmenu{
+        grid-area: topmenu;
+        /* background-color: aquamarine; */
+        display: grid;
+        grid-template-rows: repeat(auto-fit, 50px);
+        justify-content: left;
+        position: fixed;
+    
+    }
+    button>p{
+        display: none;
+    }
+    .website{
+        display: grid;
+        justify-content: center;
+        grid-template-columns:  50px 620px 320px ;
+        grid-template-rows:  720px 1fr 850px ;
+        grid-template-areas: 
+        "leftbar middlebar rightbar";
+    
+    }
+    .topmenu > .tweet-button > button{
+        height: 46px;
+        width: 45px;
+        background-color: rgb(29, 155, 240);
+        color: white;
+        font-size: 17px;
+        display: block;
+        text-align: center;
+    }
+    .topmenu > .tweet-button > button >svg{
+        display: block;
+        fill: white;
+    }
+    .bottommenu > button{
+        width: 65px;
+        height: 65px;
+        display: grid;
+        grid-template-columns: 50px ;
+        border-radius:50px;
+        border: 0px;
+        background-color: white;
+        /* justify-content: center; */
+        align-content: center;
+        padding-left: 5px;
+    }
+    .bottommenu > button> svg{
+        display: none;
+    }
+}
+@media screen and (max-width: 970px){
+   .rightbar{
+    display: none;
+   }
+ .website{
+    padding-left: 100px;
+ }
+}
+@media screen and (max-width: 900px){
+    .leftbar{
+        display: none;
+    }
+    .website{
+        display: grid;
+        justify-content: center;
+        grid-template-columns:620px;
+        grid-template-rows:  720px 1fr 850px ;
+        grid-template-areas: 
+        "middlebar";
```

Comment:
Nice long code! Mine was also extremely long like yours. It feels good to see that other people have also long lines of code.